meta-resin-common/recipes-support/resin-device-pinning/resin-device-pinning/resin-device-pinning

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+LOG_FILE=/var/log/resin-device-pinning-curl.log
+PINNING_CONF=/etc/resin-device-pinning.conf
```

Comment:
What is this used for?

Reply:
The file contains the RELEASE_ID to pin

Reply:
There is no reference to this process anywhere. Basically this PR doesn't describe anywhere how this would work.